llvm/test/CodeGen/RISCV/inline-asm-S-constraint.ll

```diff
@@ -3,6 +3,7 @@
 ; RUN: llc -mtriple=riscv64 < %s | FileCheck %s --check-prefix=RV64
 
 @var = external dso_local global i32, align 4
+@a = external global [2 x [2 x i32]], align 4
```

Comment:
Did you mean to change this file in this PR? I think this change has already been made in #80201

Reply:
Sorry, no. I accidentally included this when preparing tests using my experience on x86 Ws and RISC-V S:)